docs/src/usage.txt

```diff
@@ -1,30 +1,27 @@
-tealdeer 1.6.1
+tealdeer 1.2.3: A fast TLDR client
 Danilo Bargen <mail@dbrgn.ch>, Niklas Mohrin <dev@niklasmohrin.de>
-A fast TLDR client
```

Comment:
The default help output in clap now doesn't list version and author anymore.

I updated the help template to include those two again. The version is now combined with the description to get more compact output.